### ISSUE TITLE
`./gradlew dist -PkotlinProjectPath= ` doesn't work

### ISSUE THREAD

**Issue by wuseal** [JetBrains/kotlin-native#3300]
```
android@android-srv:~/Kotlin/Native/kotlin-native$ ./gradlew dist -PkotlinProjectPath="../Demo"

FAILURE: Build failed with an exception.

* Where:
Settings file '/home/android/Kotlin/Native/kotlin-native/settings.gradle' line: 54

* What went wrong:
Project :include:kotlin-compiler not found.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

**Comment by SvyatoslavScherbina** [comment]
According to [readme](https://github.com/JetBrains/kotlin-native/blob/master/README.md), `kotlinProjectPath` is used to enable composite build with [Kotlin project](https://github.com/JetBrains/kotlin), and the value of this property should be the path to sources of Kotlin project (e.g. obtained by cloning the repository referred above).

Does "../Demo" directory contain sources from Kotlin repository?

**Comment by wuseal** [comment]
@SvyatoslavScherbina No, It's a Kotlin/Native Demo APP Project saying `Hello, World`, And I May misunderstood the `composite build`, If there any detail DOC for this?

**Comment by SvyatoslavScherbina** [comment]
> No, It's a Kotlin/Native Demo APP Project saying `Hello, World`

Then what are you trying to achieve?

> If there any detail DOC for that?

For what?

**Comment by wuseal** [comment]
> Then what are you trying to achieve?
Just want to have a try with Kotlin/Native building  follow the `Readme`
> For what?
Just to get some information about it, And if possible, I want to make some contributions for `K/N` or `Kotlin` 😄

**Comment by SvyatoslavScherbina** [comment]
> Just want to have a try with Kotlin/Native building follow the Readme

Readme is detailed enough for this.

> Just to get some information about it, And if possible, I want to make some contributions for K/N or Kotlin 😄

I'm asking "for what?", not "what for?".

If you try to build Kotlin/Native distribution, then 
1. what do you need composite build for?
2. why are you trying to build it along with your demo app?

**Comment by wuseal** [comment]
>1. what do you need composite build for?

Not any definite goal, And What I can do with `composite build`? I am newer On K/N, Just want to known more about all the things as I can

> 2. why are you trying to build it along with your demo app?

I think I misunderstood the function of `composite build`, I just only want to build the demo with K/N.

Not Any problem now. I Close this issue.

**Comment by SvyatoslavScherbina** [comment]
> Not any definite goal, And What I can do with composite build?

According to [readme](https://github.com/JetBrains/kotlin-native/blob/master/README.md), you can build Kotlin/Native and Kotlin in single build, which is required if you make some changes to both of this projects and want to test these in conjunction.

> I just only want to build the demo with K/N.

Then you don't need to build K/N itself for this, just use most recent published release (1.3.50 currently).
Please find more details in the documentation, e.g.: https://kotlinlang.org/docs/reference/native-overview.html#how-to-start

**Comment by wuseal** [comment]
@SvyatoslavScherbina Great! I Got it!! Very thanks for your answer, Answered my problem perfectly!
